app/views/curation_concerns/base/_form_representative_image.html.erb

```diff
@@ -6,7 +6,7 @@
           Assign a Thumbnail
         </legend>
         <p>Select the file whose thumbnail should represent this <%= curation_concern.human_readable_type %>.</p>
-        <%= f.select( :representative, Hash[ curation_concern.generic_files.map {|file| [file.title.first, file.id] } ],{}, { class: 'form-control' } ) %>
+        <%= f.select( :representative, Hash[ @presenter.file_presenters.map {|file| [file.to_s, file.id] } ],{}, { class: 'form-control' } ) %>
```

Comment:
I think we should call it `@form` and make it have the same interface as the presenter.  But instead of backing by a SolrDoc, give it a Fedora object